lua/autorun/server/cfc_block_luarun.lua

```diff
@@ -2,4 +2,4 @@ hook.Add( "AcceptInput", "CFC_BlockLuaRun", function( ent )
     if ent:GetClass() == "lua_run" then
         return true
     end
-end )
+end )
```

Comment:
Make sure your file has a newline at the end!

Reply:
👍